language-server/modules/langserver-core/src/test/resources/codeaction/getter-setter/config/getterSetterCodeAction5.json

```diff
@@ -0,0 +1,11 @@
+{
+  "position": {
+    "line": 2,
+    "character": 17
+  },
+  "source": "getterSetter5.bal",
+
+  "expected": [
+
+  ]
```

Comment:
```suggestion
  "expected": []
```

Reply:
Updated in 74a8579fa2abf778fd506a18a96a4ff6239865fc